CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### ✅ Added
 - The SDK now provides a CallKit integration out of the box. [#334](https://github.com/GetStream/stream-video-swift/pull/334)
+- Infrastructure required for out-of-the-box NoiseCancellation support [#353](https://github.com/GetStream/stream-video-swift/pull/353)
```

Comment:
would remove the "out-of-the-box", it's not really out of the box here, but leaving it up to you.

Reply:
Once the feature is complete though it will be out of the box, right?

Reply:
technically you still need to bring sth which is outside `StreamVideo`, but as mentioned, no strong opinions here. If you decide to keep it, please keep the two entries "out of the box" consistent.